### PR TITLE
Zabbix host set via variable.

### PR DESCRIPTION
**Pull request by Katka92**
Now results are sent to different zabbix hosts.

### CODE REVIEW

File: che-start-workspace/jenkins-job/che-start-workspace.yml [hunk 1]
```diff
@@ -134,23 +135,28 @@
...
+          artifacts: 'che-start-workspace/*.log,che-start-workspace/*.png,che-start-workspace/*.csv,che-start-workspace/*.html'
+          allow-empty: 'true'
+          fingerprint: true
 
-- job:
+- job-template:
     name: 'che-start-workspace'
     project-type: freestyle
     node: osioperf-server-1-centos7-x86_64
-    wrappers:
```
Comment (ScrewTSW) [inline che-start-workspace/jenkins-job/che-start-workspace.yml]: Nezabudla si tie wrappery pridat naspat? :D

File: che-start-workspace/jenkins-job/che-start-workspace.yml [hunk 2]
```diff
@@ -134,23 +135,28 @@
...
-        - che-functional-perf-test-template-wrapper-prod-2
     <<: *che-functional-perf-test-template
 
-- job:
+- job-template:
     name: 'che-start-workspace-2a'
     project-type: freestyle
     node: osioperf-server-1-centos7-x86_64
-    wrappers:
```
Comment (ScrewTSW) [inline che-start-workspace/jenkins-job/che-start-workspace.yml]: tu tiez?

File: che-start-workspace/jenkins-job/che-start-workspace.yml [hunk 3]
```diff
@@ -121,6 +121,7 @@
     triggers:
         - timed: H 1 * * *
     wrappers:
+        - che-functional-perf-test-template-wrapper-{cluster}
```
Comment (Katka92) [inline che-start-workspace/jenkins-job/che-start-workspace.yml]: I moved wrappers here to be able to use parameters easily.

Comment (ScrewTSW) [reply che-start-workspace/jenkins-job/che-start-workspace.yml]: Oh, ok. Nice :)

**Pull request merged**